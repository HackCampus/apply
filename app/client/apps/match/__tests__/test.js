const test = require('ava')
test.todo('match')
