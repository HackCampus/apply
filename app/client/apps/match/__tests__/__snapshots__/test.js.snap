exports[`init 1`] = `
Object {
  "effect": Array [
    Object {
      "payload": undefined,
      "type": "fetchApplications",
    },
  ],
  "model": Object {
    "applications": Array [],
    "children": Object {},
    "detailRow": null,
    "orderBy": Object {
      "column": "createdAt",
      "direction": true,
    },
    "ordering": Array [],
    "view": "list",
  },
}
`;

exports[`view - initial 1`] = `
"<div class="matching">
        <div class="header">
        <h2>HackCampus matching - 2017</h2>
        <p>0 finished / 0 total
      </p>
    </div>
        <div class="listView">
        <table>
          <tr>
            <th>Name</th><th>University</th><th>Gender</th><th>Created ðŸ”¼</th><th>Finished</th>
          </tr>
          
        </table>
      </div>
      </div>"
`;
